### PR TITLE
Updates for autoscaling

### PR DESCRIPTION
**Pull request by danieltprice**
Updates for Autoscaling:
- Create your first project:
https://neon-next-git-dprice-autoscaling-updates-neondatabase.vercel.app/docs/get-started-with-neon/setting-up-a-project
- Compute size configuration (Create a project)
https://neon-next-git-dprice-autoscaling-updates-neondatabase.vercel.app/docs/manage/projects#compute-size-configuration
- Compute size configuration (Edit a compute endpoint)
https://neon-next-git-dprice-autoscaling-updates-neondatabase.vercel.app/docs/manage/endpoints#compute-size-configuration
- Various changes to create project steps in other topics

### CODE REVIEW

File: content/docs/get-started-with-neon/setting-up-a-project.md [hunk 1]
```diff
@@ -10,11 +10,13 @@ To create your first project:
 
 ![Create a project](/docs/get-started-with-neon/create_project.png)
 
-3. On the **Project Creation** dialog, Specify a name, a PostgreSQL version, and a region, and click **Create project**.
+3. On the **Project Creation** dialog, specify a name, a PostgreSQL version, and a region. If you are a [Neon paid plan](https://neon.tech/docs/introduction/billing#neon-plans) user, you can also specify compute size settings. Neon supports fixed size computes and Autoscaling. For more information about these options, see [Compute size configuration](/docs/manage/projects#compute-size-configuration).
```
Comment (duskpoet) [inline content/docs/get-started-with-neon/setting-up-a-project.md:13]:
```suggestion
3. On the **Project Creation** dialog, specify a name, a PostgreSQL version, and a region. If you are a [Neon paid plan](https://neon.tech/docs/introduction/billing#neon-plans) user, you can also specify compute size settings. Neon supports fixed size computes and Autoscaling (beta). For more information about these options, see [Compute size configuration](/docs/manage/projects#compute-size-configuration).
```

File: content/docs/manage/projects.md [hunk 2]
```diff
@@ -17,16 +17,28 @@ A Neon project is created with the following resources, by default:
...
+3. Specify a name, a PostgreSQL version, and a region. If you are a Neon [paid plan](/docs/introduction/billing#neon-plans) user, you can also specify compute size settings. Neon supports fixed size computes and _Autoscaling_. For more information, see [Compute size configuration](#compute-size-configuration).
+4. Click **Create Project**.
 
 Upon creating a project, you are presented with a dialog that provides your connection details for the default `neondb` database, including your password.
 
+## Compute size configuration
+
+Neon [paid plan](/docs/introduction/billing#neon-plans) users can configure compute size settings when [creating a project](#create-a-project).
+
+_Compute size_ is the number of Compute Units (CUs) assigned to a Neon compute endpoint. The number of CUs determines the processing capacity of the compute endpoint. One CU is equal to 1 vCPU with 4 GBs of RAM. Currently, a Neon compute endpoint can have anywhere from .25 CUs to 3 CUs. Larger compute sizes will be supported in a future release.
```
Comment (duskpoet) [inline content/docs/manage/projects.md:35]:
```suggestion
_Compute size_ is the number of Compute Units (CUs) assigned to a Neon compute endpoint. The number of CUs determines the processing capacity of the compute endpoint. One CU is equal to 1 vCPU with 4 GBs of RAM. Currently, a Neon compute endpoint can have anywhere from .25 CUs to 7 CUs. Larger compute sizes will be supported in a future release.
```

File: content/docs/manage/projects.md [hunk 3]
```diff
@@ -17,16 +17,28 @@ A Neon project is created with the following resources, by default:
...
+
+Neon supports two compute size configuration options:
+
+- **Fixed Size:** This option allows you to select a fixed compute size ranging from .25 CUs to 3 CUs. A fixed-size compute does not scale to meet workload demand.
```
Comment (duskpoet) [inline content/docs/manage/projects.md:39]:
```suggestion
- **Fixed Size:** This option allows you to select a fixed compute size ranging from .25 CUs to 7 CUs. A fixed-size compute does not scale to meet workload demand.
```

File: content/docs/manage/projects.md [hunk 4]
```diff
@@ -17,16 +17,28 @@ A Neon project is created with the following resources, by default:
...
+- **Autoscaling:** This option allows you to specify a minimum and maximum compute size. Neon scales the compute size up and down within the selected compute size boundaries to meet workload demand. Currently, _Autoscaling_ supports a range of 1 to 3 CUs. The 1/4 CU and 1/2 CU settings, called _shared compute_, will be supported with _Autoscaling_ in a future release.
```
Comment (duskpoet) [inline content/docs/manage/projects.md:40]: I think it's worth to link this blog post for users who are curious how autoscaling works https://neon.tech/blog/postgres-autoscaling

**Review by duskpoet — APPROVED** [review]

File: content/docs/manage/projects.md (hunk 4 above)
Comment (danieltprice) [reply content/docs/manage/projects.md]: Added!

**Comment by duskpoet** [comment]
Please take the screenshot from production, and update 3CUs to 7, that's the current upper limit on prod. Or refrain from pointing it out at all, cause this number will soon change and it may change again in the future

**Comment by danieltprice** [comment]
> Please take the screenshot from production, and update 3CUs to 7, that's the current upper limit on prod. Or refrain from pointing it out at all, cause this number will soon change and it may change again in the future

I have replaced the "Pro" screen capture with the Free Tier one, but left information about the compute size options for paid plan users. I also updated content that mentions the max number for CUs.

File: content/docs/manage/projects.md [hunk 5]
```diff
@@ -17,16 +17,28 @@ A Neon project is created with the following resources, by default:
...
+
+_Compute size_ is the number of Compute Units (CUs) assigned to a Neon compute endpoint. The number of CUs determines the processing capacity of the compute endpoint. One CU is equal to 1 vCPU with 4 GBs of RAM. Currently, a Neon compute endpoint can have anywhere from .25 CUs to 7 CUs. Larger compute sizes will be supported in a future release.
+
+Neon supports two compute size configuration options:
+
+- **Fixed Size:** This option allows you to select a fixed compute size ranging from .25 CUs to 7 CUs. A fixed-size compute does not scale to meet workload demand.
+- **Autoscaling:** This option allows you to specify a minimum and maximum compute size. Neon scales the compute size up and down within the selected compute size boundaries to meet workload demand. Currently, _Autoscaling_ supports a range of 1 to 3 CUs. The 1/4 CU and 1/2 CU settings, called _shared compute_, will be supported with _Autoscaling_ in a future release. For more information about Neon's autoscaling feature, see [Scaling serverless Postgres: How we implement autoscaling](https://neon.tech/blog/postgres-autoscaling).
```
Comment (duskpoet) [inline content/docs/manage/projects.md:40]:
```suggestion
- **Autoscaling:** This option allows you to specify a minimum and maximum compute size. Neon scales the compute size up and down within the selected compute size boundaries to meet workload demand. Currently, _Autoscaling_ supports a range of 1 to 7 CUs. The 1/4 CU and 1/2 CU settings, called _shared compute_, will be supported with _Autoscaling_ in a future release. For more information about Neon's autoscaling feature, see [Scaling serverless Postgres: How we implement autoscaling](https://neon.tech/blog/postgres-autoscaling).
```

**Review by duskpoet — APPROVED** [review]

**Review by duskpoet — APPROVED** [review]

**Pull request merged**